com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/Non-CanvasSlate.shader

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-Shader "Non-CanvasSlate" {
+Shader "Graphics Tools/Non-Canvas/Slate" {
```

Comment:
Any preference for Slate vs SlateContent for the name?

Reply:
Slate is cool by me if this is _the_ slate shader. We have a [slate proximity ](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/main/com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/Non-CanvasSlateProximity.shader)shader, but that is just for drawing proximity shadows.